<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-49GARL5Q" failures="1" tests="2" name="Test" time="19.678" errors="0" timestamp="2023-02-15T12:15:03 IST">
  <testcase classname="com.selenium.amazon_automation.test.LoginTest" name="invalidLoginTest" time="12.97"/>
  <testcase classname="com.selenium.amazon_automation.test.LoginTest" name="invalidLoginTest" time="19.482">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 2 out of bounds for length 2">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 2
at com.selenium.amazon_automation.util.ProductTestListener.logTestData(ProductTestListener.java:59)
at com.selenium.amazon_automation.util.ProductTestListener.onTestSuccess(ProductTestListener.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- invalidLoginTest -->
</testsuite> <!-- Test -->
